packages.adoc

```diff
@@ -53,6 +53,13 @@ default.
 Installing the {product-short} Proxy is required as of
 {product-abbrev} 8.7.
 
+[NOTE]
+By default {product-short} Proxy is configured to enforce strict server name enforcement
```

Comment:
"enforce strict server name enforcement" somehow does not sound good.

Reply:
possible alternative: ... configured to perform strict server name enforcement

TBD: Do we also want to mention the HTTP Host header here? Perhaps that's overkill since we cover it in the admin guide, but I fear that it is non-obvious what exactly we're being strict about without that extra detail.